README.md

```diff
@@ -42,65 +44,84 @@ postgresql_pg_hba_conf:
     auth_method: ident
 ```
 ### postgresql_server_conf
-Usage of `postgresql_server_conf` adds defined values at the end of postgresql.conf.
-So the default ones are overwritten.
+The content of the `postgresql_server_conf` variable is added to the end of 
+the `/var/lib/pgsql/data/postgresql.conf` file. As a result, the default 
+settings are overwritten.
 ```yaml
 postgresql_server_conf:
   ssl: on
   shared_buffers: 128 MB
   huge_pages: try
 ```
 ### postgresql_ssl_enable
-To set up ssl connection it's necessary to set up `postgresql_ssl_enable` variable and provide server certificate and key.
+To set up a SSL/TLS connection, set the `postgresql_ssl_enable` variable to 
+`True`  and provide a server certificate and a private key.
 ```yaml
 postgresql_ssl_enable: true
 ```
 ### postgresql_cert_name
-To specify certificate name use `postgresql_cert_name` variable.
-You can copy your certificate to `/etc/pki/tls/certs/server.crt` and key to `/etc/pki/tls/private/server.key` or
-you can also use certificate system role. For more detail see [`examples/`](examples).
-```yaml
-postgresql_cert_name: server
-```
-### postgresql_key_path
-Optionaly you can specify path to server key using `postgresql_key_path` variable. The default value is
+If you want to use your own certificate and private key, use the 
+`postgresql_cert_name` variable to specify the certificate name. You must keep 
+both certificate and key files in the same directory and under the same name 
+with the `.crt` and `.key` suffixes.
+
+For example, if your certificate file is located in `/etc/certs/server.crt` and 
```

Comment:
Will the role ensure correct permissions on key and cert files or the user must do this?